app-web/forgejo/autobuild/defines

```diff
@@ -9,5 +9,7 @@ BUILDDEP="go nodejs"
 # FIXME: Autobuild does not yet support splitting out debug symbol from Go executables.
 ABSPLITDBG=0
 
-# FIXME: runtime.cgo_yield: relocation target _cgo_yield not defined
-FAIL_ARCH="riscv64"
+# FIXME: riscv: runtime.cgo_yield: relocation target _cgo_yield not defined
+# FIXME: loongson3: nodejs broken
+#        https://github.com/nodejs/node/issues/56973
+FAIL_ARCH="@(riscv64|loongson3)"
```

Comment:
Oops, I missed this one, it should be  `FAIL_ARCH="!(riscv64|loongson3)"`

Reply:
Followed-up in https://github.com/AOSC-Dev/aosc-os-abbs/pull/9619